examples/getting-started-movielens/03-Training-with-TF.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>May be explain what's DCN briefly and add the image here?  you can steal information from this notebook :)  https://github.com/NVIDIA-Merlin/models/blob/main/examples/03-Exploring-different-models.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>explanation + image added! 🙂</p>